LiskHQ/lisk-hub
# Add vote transaction details - Closes #704

### What was the problem?
No details of voted delegates in `Vote` transaction
### How did I fix it?
Implemented voters details
### How to test it?
Go to wallet, click on a transaction with `Vote` type and check if voters appear.
### Review checklist
- The PR solves #704
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
